shared/chroot.go

```diff
@@ -221,6 +221,9 @@ func SetupChroot(rootfs string, envs DefinitionEnv) (func() error, error) {
 				Value: "noninteractive",
 				Set:   true,
 			},
+			"TMPDIR": EnvVariable{
+				Set: false,
+			},
```

Comment:
Instead of hard-coding this, you could just override `TMPDIR` in the yaml file:
```yaml
environment:
  variables:
    - key: TMPDIR
      value: ""
```

Reply:
ALT Linux apt does not support --auto-remove option to remove command, so the current apt manager could not be used.

Reply:
I see. In that case please use the custom manager like [here](https://github.com/lxc/distrobuilder/blob/master/doc/examples/plamolinux#L52-L62).